### PR TITLE
add dialog

### PR DESCRIPTION
**Pull request by artozah**

### CODE REVIEW

File: _client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  		</paper-action-dialog>
+  	</section>	
+	</template>
+	<script>
+		Polymer('iwish-dialog', {
+      toggleDialog: function(e) {
+        if (e.target.localName != 'button') {
+          return;
+        }
+        var d = e.target.nextElementSibling;
```
Comment (patyatka) [inline _client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html]: more descriptive please. dialog is better then just b

File: _client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	  			<core-item icon="settings" label="word or phrase"></core-item>
+	  			<paper-input></paper-input>
+	  		</div>
+    		<paper-button affirmative raised class="colored">Done</paper-button>
+  		</paper-action-dialog>
+  	</section>	
+	</template>
+	<script>
+		Polymer('iwish-dialog', {
+      toggleDialog: function(e) {
```
Comment (patyatka) [inline _client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html]: there is problem with identations

File: _client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html (hunk 1 above)
Comment (romanfromrome) [inline _client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html]: agreed) no 'e' and 'd' vars please)

File: _client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<link rel="import" href="../../bower_components/paper-dialog/paper-action-dialog.html">
+<link rel="import" href="../../bower_components/paper-radio-group/paper-radio-group.html">
+<link rel="import" href="../../bower_components/core-item/core-item.html">
+
+<polymer-element name="iwish-dialog" atributes="">
+	<template>
+		<section on-tap="{{toggleDialog}}">
```
Comment (romanfromrome) [inline _client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html]: better move(or bind) this to external attribute, so we would be able to toggle it externally from parent

File: _client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        if (e.target.localName != 'button') {
+          return;
+        }
+        var d = e.target.nextElementSibling;
+        if (!d) {
+          return;
+        }
+        d.toggle();
+      }
+		})
```
Comment (romanfromrome) [inline _client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html]: I guess we need to configure jshint to look through html's as well. Add ```;``` please + ```'use strict'``` (*yes, i will add those to my components as well)

File: _client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
+<link rel="import" href="../../bower_components/paper-dialog/paper-action-dialog.html">
+<link rel="import" href="../../bower_components/paper-radio-group/paper-radio-group.html">
+<link rel="import" href="../../bower_components/core-item/core-item.html">
+<link rel="import" href="../../bower_components/core-icon-button/core-icon-button.html">
+
+<polymer-element name="iwish-dialog" atributes="">
+	<template>
+		<div on-tap="{{toggleDialog}}">
```
Comment (romanfromrome) [inline _client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html]: can't see a change here. let's go ahead and discuss this tomorrow

File: _client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+	  			<paper-input layout></paper-input>
+	  		</div>
+    		<paper-button id="done" affirmative raised class="colored">done</paper-button>
+  		</paper-action-dialog>
+  	</div>	
+	</template>
+	<script>
+
+		Polymer('iwish-dialog', {
+      toggleDialog: function(event) {
```
Comment (romanfromrome) [inline _client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html]: indents here and throught the whole object - are missed up

**Pull request merged**